template-transformer/handler.lua

```diff
@@ -162,13 +162,15 @@ function TemplateTransformerHandler:body_filter(config)
         if body == nil then
           return ngx.ERROR
         end
+        local query_string = req_get_uri_args()
```

Comment:
Since there is a specific context here (response), maybe we should explicitly say that we are exposing "req_query_string" to the context so that it's is easier to understand what's going on. Than we change the usage on the response template itself. What do you think?

Reply:
Makes sense to me!